pkg/scheduler/framework/plugins/imagelocality/image_locality.go

```diff
@@ -80,14 +80,16 @@ func New(_ runtime.Object, h framework.FrameworkHandle) (framework.Plugin, error
 
 // calculatePriority returns the priority of a node. Given the sumScores of requested images on the node, the node's
 // priority is obtained by scaling the maximum priority value with a ratio proportional to the sumScores.
-func calculatePriority(sumScores int64) int64 {
-	if sumScores < minThreshold {
-		sumScores = minThreshold
-	} else if sumScores > maxThreshold {
-		sumScores = maxThreshold
+func calculatePriority(sumScores int64, sizeContainers int) int64 {
+	minThresholds := minThreshold * int64(sizeContainers)
```

Comment:
Do we need to scale the minimum threshold as well? I see the case for scaling the max threshold (to get a more accurate score by making it less likely for large pods to normalize to the max threshold), but having more containers in an image doesn't affect the chance that `sumScores` will be less than `minThreshold`. With this change I think you would actually see more scores being normalized toward the new minThreshold

Reply:
+1, agree with you!

Reply:
Go with the normalization per image, the thresholds for image has not been changed,  the old logic still applicable.